docs/classes/models/xmod.rst

```diff
@@ -0,0 +1,23 @@
+X-MOD
+=====
+
+.. note::
+    The X-MOD implementation integrated into Transformers already supports adapters.
+    To make this implementation compatible with Adapters, a few changes were necessary:
+
+        - In Adapters, the X-MOD classes rely on the usual adapter methods instead of the custom methods introduced in Transformers, i.e.:
+            - ``set_active_adapters()`` instead of ``set_default_language()``.
+            - ``AdapterSetup`` context instead of ``lang_ids`` parameter.
+        - We provide dedicated model checkpoints converted for usage with Adapters
```

Comment:
I think this should make it more clear that the original checkpoints do not work with the `adapters` model.

Reply:
Tried to make it a bit clearer.